docs/source/contribution/cpu_features.rst

```diff
@@ -0,0 +1,214 @@
+.. ******************************************************************************
+.. * Copyright contributors to the oneDAL project
```

Comment:
Question: I see other files always mention the year in the copyright notice - for example:
https://oneapi-src.github.io/oneDAL/examples/cpp/basic_statistics_dense_batch.html

Isn't it required to have it?